ocs_ci/ocs/cluster.py

```diff
@@ -912,6 +913,85 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+    :return: dict returned from cmd
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
+
+def validate_replica_data(pool_name, replica):
+    """
+    Check if data is replica 2 or 3
+    :param replica: size of the replica(2,3)
+    :type replica: int
+    :param pool_name: name of the pool to check replica
+    :type pool_name: str
+    :return: true if replica is verified, false if not
+    :rtype: bool
```

Comment:
I see no change in the docstring. What we expect would look like:
```
Args:
    replica (int): size of the replica (eg. 2 or 3)
    pool_name (str): name of the pool to check replica

Returns:
    true if replica is verified, false if not
```

Reply:
See [Napoleon - Marching toward legible docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/) for few examples.